apps/agreement/contracts/Agreement.sol

```diff
@@ -16,9 +16,10 @@ import "@aragon/os/contracts/lib/math/SafeMath.sol";
 import "@aragon/os/contracts/lib/math/SafeMath64.sol";
 import "@aragon/staking/contracts/Staking.sol";
 import "@aragon/staking/contracts/StakingFactory.sol";
+import "@aragon/staking/contracts/locking/ILockManager.sol";
 
 
-contract Agreement is IAgreement, AragonApp {
+contract Agreement is IAgreement, AragonApp, ILockManager {
```

Comment:
Minor nitpick, but can we move `ILockManager` to be first?

It seems fairly well isolated and independent from the other two, which are a lot more heavy.

Reply:
Sure, I’ll try. Do we have a standard criteria for this?